samples/e2e/EventHubsCaptureEventGridDemo/FunctionEGDWDumper/Function1.cs

```diff
@@ -66,14 +66,18 @@ public static void Run([EventGridTrigger]JObject eventGridEvent, TraceWriter log
         private static async void Dump(Uri fileUri)
         {
             // Get the blob reference
-            var storageAccount = CloudStorageAccount.Parse(StorageConnectionString);
-            var blobClient = storageAccount.CreateCloudBlobClient();
-            var blob = await blobClient.GetBlobReferenceFromServerAsync(fileUri);
+            string connectionString = "ConnectionString";
+            string containerName = "containerName";
+            string blobName = "blobName";
```

Comment:
Shouldnt we be using `StorageConnectionString` and `fileUri`?

Reply:
I missed these defined parameter。I have updated it.